geometry_msgs/Point goal_position
float32[] goal_joints
---
bool goal_achieved
---
bool active
